sycl/test/usm/math.cpp

```diff
@@ -0,0 +1,81 @@
+// RUN: %clangxx -fsycl %s -o %t.out -lOpenCL
+// RUN: %CPU_RUN_PLACEHOLDER %t.out
```

Comment:
Does this functionality work with host device? If yes, please add run on host.
`// RUN: env SYCL_DEVICE_TYPE=HOST %t.out`

Reply:
I copied the test environments from other USM tests, which only test the CPU_RUN_PLACEHOLDER environment. I can try the host device and add it if it works.

Reply:
I did not work with USM, but I believe that it should not break the work with the host device, and therefore should be tested.

Reply:
I tested it with the host device, and it did fail. James says that the host device doesn't support it at the moment.